WARNING:root:Too few points to create valid contours
/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py:545: UserWarning: No labelled objects found. Use label='...' kwarg on individual plots.
  warnings.warn("No labelled objects found. "
WARNING:root:Too few points to create valid contours
('800', '900', 'normal')
Running object 800
0.0
{'i': 84, 'r': 76, 'z': 166, 'g': 55}
{'i': 7.4207363, 'r': 3.3461108, 'z': 10.505102, 'g': 0.6208403}
Variable??
4.630474908574085
Non-Variable source in g
Variable??
6.970098489524835
Variable Source!
Variable??
14.059542004544328
Variable Source!
Variable??
15.631318965880565
Variable Source!
Slope:       [1, 1, 0.6857884899516234, 0.7383084396064794]
Slope Error: [0.44660732907692696, 0, 0.05763956494117613, 0.054610527591862204]
intercepts:  [0.04153256605836342, 0, -0.010974884631983357, -0.008144108759699776]
Int Error:   [0.022736134584842634, 0, 0.0028744195895748746, 0.0026893736424647764]
StDev:       [0.101568826, 0.16384573, 0.3409682, 0.53493875]
('ROW:', 800, 'Tau:', '3.163163333263922e-05', 'V:', '-0.8005825670332379')
Running object 801
0.0
{'i': 83, 'r': 76, 'z': 161, 'g': 72}
{'i': 2.4919724, 'r': 2.141895, 'z': 2.655058, 'g': 1.316935}
Variable??
4.3963557934126
Non-Variable source in g
Variable??
3.804259884542807
Non-Variable source in r
Variable??
3.036434601228966
Non-Variable source in i
Variable??
3.6168409296847086
Non-Variable source in z
Slope:       [0.8706032277432554, 1, 1, 1]
Slope Error: [0.21362744764682343, 0, 0.08036920704478595, 0.10617818537829501]
intercepts:  [0.0028698459995619707, 0, 0.0047452001443449, -0.0007197860969823111]
Int Error:   [0.008729699348198162, 0, 0.00333357693086748, 0.004110766908053435]
StDev:       [0.10409971, 0.09070182, 0.0773274, 0.13719079]
('ROW:', 801, 'Tau:', '2.8095288244601058', 'V:', '-2.178033355862831')
Running object 802
0.0
{'i': 83, 'r': 70, 'z': 166, 'g': 9}
{'i': 2.0943542, 'r': 0.90041053, 'z': 2.9597387, 'g': 0.24452223}
Variable??
1.7090080926815667
Non-Variable source in g
Variable??
2.315352527446419
Non-Variable source in r
Variable??
2.501553372557782
Non-Variable source in i
Variable??
3.0948770661673097
Non-Variable source in z
Slope:       [1, 1, 1, 1]
Slope Error: [0.7156200656974975, 0, 0.05765351910885248, 0.04658036934518812]
intercepts:  [-0.07276896411475803, 0, -0.006052898242588593, 0.008067102690262179]
Int Error:   [0.026607830510793413, 0, 0.0046860564428876815, 0.0038286891637818194]
StDev:       [0.01856875, 0.07268085, 0.0786529, 0.14302698]
('ROW:', 802, 'Tau:', '2.6009350912211233', 'V:', '-2.075022133276761')
Running object 803
0.0
{'i': 78, 'r': 72, 'z': 142, 'g': 50}
{'i': 1.1912583, 'r': 1.0166538, 'z': 1.272419, 'g': 0.54775834}
Variable??
2.2114351098898286
Non-Variable source in g
Variable??
3.4047440576367634
Non-Variable source in r
Variable??
4.398145241998314
Non-Variable source in i
Variable??
3.4685187101469754
Non-Variable source in z
Slope:       [0.6522747241642484, 1, 0.40477956777866186, 0.6034592675551178]
Slope Error: [0.14942184370501374, 0, 0.11645081878576069, 0.10833137921452524]
intercepts:  [0.0010944375527082162, 0, -0.006526010739978916, 0.003438239116887787]
Int Error:   [0.013223995185079374, 0, 0.011175584945204271, 0.010143266480846137]
StDev:       [0.05698053, 0.097652175, 0.1283054, 0.16046217]
('ROW:', 803, 'Tau:', '0.00014435895924208575', 'V:', '-0.7342636568807626')
Running object 804
0.0
{'i': 71, 'r': 28, 'z': 140}
{'i': 0.93090296, 'r': 0.39293608, 'z': 1.4488256, 'g': 0.0}
Variable??
0
Non-Variable source in g
Variable??
2.032989501093322
Non-Variable source in r
Variable??
2.184402232682129
Non-Variable source in i
Variable??
2.5602029136684274
Non-Variable source in z
Slope:       [1, 1, 0.26083648253536884, 0.32026943134911146]
Slope Error: [0.0, 0, 0.1263020172724834, 0.07781758306935427]
intercepts:  [0, 0, -0.029989501742977875, -0.01735444330096092]
Int Error:   [0.0, 0, 0.018927391166284, 0.011156467903016918]
StDev:       [0.0, 0.05802104, 0.100392975, 0.17649487]
('ROW:', 804, 'Tau:', '2.925146182362979e-05', 'V:', '0.26659427005980774')
Traceback (most recent call last):
  File "ClusterEmcee2_linear_mu_2step.py", line 509, in <module>
    perform_emcee_step2(time, flux_mod, err_mod, dMu_dict, scale_dict, color_sort_ones, ROW, mu)
  File "ClusterEmcee2_linear_mu_2step.py", line 339, in perform_emcee_step2
    sausageplot_step2(max_theta[0], time, flux, max_theta[1], 5, err**2, dMu_dict, scale_dict, color_sort_ones, ROW, fig)
  File "ClusterEmcee2_linear_mu_2step.py", line 178, in sausageplot_step2
    ax4.legend()
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py", line 564, in legend
    self.legend_ = mlegend.Legend(self, handles, labels, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend.py", line 386, in __init__
    self._init_legend_box(handles, labels, markerfirst)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend.py", line 655, in _init_legend_box
    fontsize, handlebox))
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend_handler.py", line 119, in legend_artist
    fontsize, handlebox.get_transform())
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend_handler.py", line 476, in create_artists
    self.update_prop(coll, barlinecols[0], legend)
IndexError: tuple index out of range
